### PR TITLE
modules/accounts unit tests

### PR DESCRIPTION
**Pull request by roman-bystrimovich**

### CODE REVIEW

File: tests/unit/modules/account.spec.js [hunk 1]
```diff
@@ -0,0 +1,1885 @@
...
+                transaction : {
+                    attachAssetType : sinon.spy()
+                }
+            } 
+        };
+        var callback = sinon.spy();
+        var account = new Accounts(callback, scope);
+
+        expect(scope.logic.transaction.attachAssetType.called).to.be.true;
+        expect(scope.logic.transaction.attachAssetType.calledOnce).to.be.true;
```
Comment (nerdvibe) [inline tests/unit/modules/account.spec.js]: Isn't calledOnce enough in this case?

File: tests/unit/modules/account.spec.js [hunk 2]
```diff
@@ -0,0 +1,1885 @@
...
+            account.getAccount({ address : testAccount.address }, testObj, callback);
+
+            expect(scope.logic.account.get.calledOnce).to.be.true;
+            expect(scope.logic.account.get.getCall(0).args.length).to.equal(3);
+            expect(scope.logic.account.get.getCall(0).args[0]).to.deep.equal({ address : testAccount.address });
+            expect(scope.logic.account.get.getCall(0).args[1]).to.equal(testObj);
+            expect(scope.logic.account.get.getCall(0).args[2]).to.equal(callback);
+        });
+
+        it("public.getAccounts", function() {
```
Comment (nerdvibe) [inline tests/unit/modules/account.spec.js]: Shouldn't this be in a outside it?

File: tests/unit/modules/account.spec.js [hunk 3]
```diff
@@ -0,0 +1,1885 @@
...
+            expect(callback.getCall(0).args[0]).to.equal(null);
+            expect(callback.getCall(0).args[1]).to.deep.equal({
+                balance : testAccount.balance,
+                unconfirmedBalance : testAccount.u_balance
+            });
+        });
+
+        it("validation passed, getAccount returns account with not empty  balance", function() {
+            testAccount.balance = '1234';
+            testAccount.u_balance = '5678'
```
Comment (nerdvibe) [inline tests/unit/modules/account.spec.js]: ;

File: tests/unit/modules/account.spec.js [hunk 4]
```diff
@@ -0,0 +1,1885 @@
...
+            expect(callback.calledOnce).to.be.true;
+            expect(callback.getCall(0).args.length).to.equal(2);
+            expect(callback.getCall(0).args[0]).to.equal(null);
+            expect(callback.getCall(0).args[1]).to.deep.equal({ delegates : [] });
+        });
+
+        it("validation passed, getAccount returns account with delegates", function() {
+            testAccount.delegates = ['1','2','3','4'];
+
+            var modules = Accounts.__get__('modules')
```
Comment (nerdvibe) [inline tests/unit/modules/account.spec.js]: ;

File: tests/unit/modules/account.spec.js [hunk 5]
```diff
@@ -0,0 +1,1885 @@
...
+            request.body.multisigAccountPublicKey = firstAccount.publicKey;
+
+            modules.accounts.getAccount.onCall(0).callsArgWith(1, null, firstAccount);
+            modules.accounts.getAccount.onCall(1).callsArgWith(1, null, secondAccount);
+            scope.logic.transaction.create.throws(error)
+
+            account.shared.addDelegates(request, callback);
+
+            expect(scope.schema.validate.calledOnce).to.be.true;
+            expect(scope.schema.v
```
Comment (nerdvibe) [inline tests/unit/modules/account.spec.js]: Indentation broken

**Review by nerdvibe — COMMENTED** [review]
Nice job on this PR @roman-bystrimovich . 👍 
I've left some minors tiny improvements, but I really liked this PR..

**Comment by roman-bystrimovich** [comment]
Fixed. Thanks @xunga for audit

**Pull request closed without merging**